### PR TITLE
docs: add angular.dev release banners

### PR DESCRIPTION
**Pull request by twerske**
v18!!!!!!!!!! wow

### CODE REVIEW

**Review by JeanMeche — APPROVED** [review]
🎆

File: adev/src/app/features/home/home.component.html [hunk 1]
```diff
@@ -2,9 +2,13 @@
   <div class="adev-top">
     <div class="adev-top-content">
       <!-- Banner(s) can be edited here! -->
-      <a href="https://goo.gle/angular-dot-dev" class="adev-banner" target="_blank">
-        <h1 tabindex="-1">Welcome to Angular.dev &#8212; the new home for Angular!</h1>
-        <p class="adev-banner-cta">Read about our launch</p>
+      <a href="https://goo.gle/angular-v18" class="adev-banner" target="_blank">
+        <h1 tabindex="-1">Angular v18 is new available!</h1>
```
Comment (crisbeto) [inline adev/src/app/features/home/home.component.html:6]: new -> now

**Review by crisbeto — DISMISSED** [review]

**Review by jelbourn — APPROVED** [review]
LGTM for content, though I just now noticed that these banners are `<h1>` elements, which I don't think is correct

**Comment by twerske** [comment]
> LGTM for content, though I just now noticed that these banners are `<h1>` elements, which I don't think is correct

A follow up PR :)

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit 1d774d6175d2c7214c84f5d1d12f3072ff916c87.

**Pull request closed without merging**